### PR TITLE
Switched to Microsoft Rewards.  

### PR DESCRIPTION
**Pull request by reganface**
I made a few changes and got this working with the new Microsoft Rewards page.  Both of my live accounts worked.  I haven't tried any facebook connected accounts.  It still needs some work to get it fully functional.  getLifetimeCredits has been commented out (for now) as it's returning a non-logged in page instead of the dashboard.  I'm not sure yet how to get the logged in view - something with passing cookies I'd imagine.

This is my first time using Python as well as contributing on GitHub.  Apologies if somethings are not best practice, just let me know!

### CODE REVIEW

**Comment by jamesalmeida** [comment]
I tried your branch and it seems to be working 👍

**Comment by duddles** [comment]
Wow, your first time using python and you solved the issue - I'm impressed!

**Comment by Shah-Parth** [comment]
Please merge this to master!

**Comment by amayer5125** [comment]
I am reviewing it. If I can get the total lifetime points working I will merge it.
I am also talking with @sealemar about making a seperate branch until we are sure everyone is switched over to Microsoft Rewards. We dont want to break Bing Rewards just because most people are on MS Rewards.

**Comment by reganface** [comment]
A separate branch is probably the best way to go until the old Bing Rewards is completely dead.

I took a better look at the page that the lifetime points method was fetching and it's a javascript form submission.  I'm going to work on that now.  This is the form that it's submitting.

    <form name="fmHF" id="fmHF" action="https://account.microsoft.com/auth/complete-silent-signin?ru=https%3a%2f%2faccount.microsoft.com%2frewards%3frefd%3daccount.microsoft.com&wa=wsignin1.0" method="post" target="_self">
        <input type="hidden" name="NAP" id="NAP" value="V%3D1.9%26E%3D1273%26C%3DTLUX57eYO2iCm9jN-H5op7DPhwEZnNuOebNStKOrEu-PD1vp18mr3w%26W%3D1">
        <input type="hidden" name="ANON" id="ANON" value="A%3D322C71A71F092E932113D6DFFFFFFFFF%26E%3D12cd%26W%3D1">
        <input type="hidden" name="t" id="t" value="GAAcAgMAAAAEgAAADAAgTR9CKu6a60VFR4sWQ7nKRLtGWRGFsrZ3u3Ct8TLXaMAAAR38spj78gBPq6uZhozAMCmeAMAwZEPTPAgZ/No2dkMQuPfc1MMtLle9HU/flgOLKr6sRh62fp2T+3kyEUfh8gdypBUoV7SfDG1OBG53v5nhaJdsnk0dQ3Ghe/liVsPlp0J11/ZR3u1c48MgFwsUeUJoIu+eYJPD7FBgl0d2+0hXStNnEg6+Junpbhe/+kGYCUimB7ydj+fgBU5IunU0rzQ9DiXP83Upuj31ty7B7C5MCgLKWuJ2eEBaYheQNJB6XyoYlu4ZbkdyxVtq4HUPaf8KL3WhoFpm26FrT1ndlL80UwO8M2gOfhLFcMR64dOge2IEbIm7gD3aLOTutHmZoFHpAHEA6QD+fwMADZT5ugSxz1cCsc9XOncEAAoQIAAQAAA5AAAAAAAAA1VTAAU5MjExNgAAAAAECQIAAHh5bUAABkEABUNocmlzAAVSZWdhbgRfAAAAAAAAAAAAAAAAkPgDVRwHZ2MAAASxz1cDWEZYAAAAAAAAAAAAAAAADQA2OC42LjI1NC4yMDEABAEAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAP0oF6ZaE42RAAAAAAAAAAAAAAAAAAAAAAAAEwBjcmVnYW4yMkBnbWFpbC5jb20AAAAAAAAAAAAAAAAAAAAAAAAA">
    </form>

**Comment by reganface** [comment]
Alright, with this new commit lifetime credits is working again!

File: pkg/bingRewards.py [hunk 1]
```diff
@@ -110,46 +110,63 @@ def requestFlyoutPage(self):
             page = helpers.getResponseBody(response)
         return page
 
+# TODO: this function is returning a page that says you must have javascript to log in
```
Comment (amayer5125) [inline pkg/bingRewards.py]: Can you delete this note?

**Comment by amayer5125** [comment]
If you delete the note with the `#TODO` and update version.txt and lines 35 & 36 in main.py I can merge this.

**Comment by sealemar** [comment]
@amayer5125 I agree with you, let's do it on a separate brunch until everybody is satisfied. Then we can merge if that won't break the existing functionality.

**Comment by reganface** [comment]
Whoops, forgot about that TODO.  That's been removed and the version has been updated

**Comment by amayer5125** [comment]
I merged this localy because I needed to resolve the merge conflict. Changes are merged.

**Comment by wipedout** [comment]
Thank you for solving for the changes to the rewards program! You folks do some good programming! Haven't learned how to use Git (Mac) to download the updates; still downloading the zip and manually changing the config file (also commented out the <events> section). Still, it's impressive that after a clean download and 3 minutes of copy/paste, the script is running once again, thanks to your updates. Thank you!^3

**Comment by Shah-Parth** [comment]
@wipedout I'm sure MAC comes with git. if it doesn't ([https://git-scm.com/book/en/v2/Getting-Started-Installing-Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)). once you do this, open the terminal and change the directory using cd to your desired folder where you would like this project to be in. Once you do that, just type `git clone https://github.com/sealemar/BingRewards.git`  and now you have the whole project. Once you have the project in your desired directory all you have to do for new releases is type this command in terminal `git pull` and you get all the new things. Hope this helps.

**Pull request closed without merging**